nx/include/switch/kernel/svc.h

```diff
@@ -597,27 +633,65 @@ Result svcGetInfo(u64* out, u32 id0, Handle handle, u64 id1);
 
 ///@}
 
+///@name Cache Management
+///@{
+
+/**
+ * @brief Flushes the entire data cache (by set/way).
+ * @note Syscall number 0x2A.
+ * @warning This is a privileged syscall. Use \ref envIsSyscallHinted to check if it is available.
+ */
+void svcFlushEntireDataCache(void);
+
+/**
+ * @brief Flushes data cache for a virtual address range.
+ * @param[in] address Address of region to flush.
+ * @param[in] size Size of region to flush.
+ * @note Syscall number 0x2B.
+ */
+Result svcFlushDataCache(void *address, size_t size);
+
+///@}
+
 ///@name Memory management
 ///@{
 
 /**
  * @brief Maps new heap memory at the desired address. [3.0.0+]
  * @return Result code.
  * @note Syscall number 0x2C.
- * @warning This is a privileged syscall. Use \ref envIsSyscallHinted to check if it is available.
  */
 Result svcMapPhysicalMemory(void *address, u64 size);
 
 /**
  * @brief Undoes the effects of \ref svcMapPhysicalMemory. [3.0.0+]
  * @return Result code.
  * @note Syscall number 0x2D.
- * @warning This is a privileged syscall. Use \ref envIsSyscallHinted to check if it is available.
  */
 Result svcUnmapPhysicalMemory(void *address, u64 size);
 
 ///@}
 
+///@name Process and thread management
+///@{
+
+/**
+ * @brief Gets information about a thread that will be scheduled in the future. [5.0.0+]
+ * @return Result code.
+ * @note Syscall number 0x2E.
+ * @warning This is a privileged syscall. Use \ref envIsSyscallHinted to check if it is available.
+ */
+Result GetDebugFutureThreadInfo(LastThreadContext *out_context, u64 *out_thread_id, Handle debug, s64 ns);
+
+/**
+ * @brief Gets information about the previously-scheduled thread.
+ * @return Result code.
+ * @note Syscall number 0x2F.
+ */
+Result GetLastThreadInfo(LastThreadContext *out_context, u64 *out_tls_address, u32 *out_flags);
```

Comment:
Missing svc prefix - also, is this really a non-privileged syscall?

Reply:
it is not privileged.